app/lib/state_change_notifier.rb

```diff
@@ -1,85 +1,77 @@
 class StateChangeNotifier
-  def self.sign_up(candidate)
-    candidate_number = Candidate.where(hide_in_reporting: false).count
-
-    return unless (candidate_number % 25).zero?
-
-    candidate_number_is_significant = (candidate_number % 100).zero?
+  APPLICATION_OUTCOME = %i[rejected declined withdrawn recruited].freeze
```

Comment:
I was confused by this constant name, that might just be me, but signifying these are event name symbols might clarify (it's also easy to confuse with application states) eg. `APPLICATION_OUTCOME_EVENTS`